compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/interop/UIKitView.uikit.kt

```diff
@@ -174,24 +80,32 @@ fun <T : UIView> UIKitView(
     interactive: Boolean = true,
     accessibilityEnabled: Boolean = true
 ) {
+    val compositeKeyHash = currentCompositeKeyHash
     val interopContainer = LocalInteropContainer.current
-    val interopViewHolder = remember {
-        UIKitViewHolder(
-            container = interopContainer,
-            createView = factory,
-            onResize = onResize,
-            onRelease = onRelease,
-            areTouchesDelayed = true
-        )
-    }
 
-    UIKitInteropLayout(
+    val backgroundColor by remember(background) { mutableStateOf(background.toUIColor()) }
+
+    InteropView(
+        factory = {
+            UIKitInteropViewHolder(
```

Comment:
Why are you not placing it inside `remember` like on Desktop?

Reply:
Because we don't need to _remember_ it, it will be bound to the `LayoutNode`, which is constructed when `[Reusable]ComposeNode` is emitted. That's the whole point of the refactor - to attach entire context of view interop (InteropViewHolder) to be managed by Compose runtime implicitly, so it can be pooled/reused, etc.

Reply:
On Desktop it's not working as intended due to `FocusSwitcher` logic requiring two separate `LayoutNode`s to be emitted with modifiers depending on the `group`. So the whole idea of `InteropViewHolder` being self-contained is not achieved for now on Desktop.